### ISSUE TITLE
Interactive Book type: Javascript error - only one instance of babel-polyfill is allowed

### ISSUE THREAD

**Issue by mikedawson** [tunapanda/h5p-standalone#52]
Hi,

I've been using h5p-standalone to run H5P's offline and it's really great for that! Unfortunately a small minority of files, in particular interactive book, seem to have a javascript conflict. If you try the [interactive book example](https://h5p.org/content-types/interactive-book) from the h5p.org website, you will get the following Javascript error on both Chrome and Firefox:

```
Uncaught Error: only one instance of babel-polyfill is allowed
    at Object.<anonymous> (h5p-interactive-book.js:1)
    at Object.<anonymous> (h5p-interactive-book.js:1)
    at n (h5p-interactive-book.js:1)
    at Module.<anonymous> (h5p-interactive-book.js:10)
    at n (h5p-interactive-book.js:1)
    at h5p-interactive-book.js:1
    at h5p-interactive-book.js:1
h5p-interactive-book.js:10 Uncaught TypeError: Cannot read property 'title' of undefined
    at new e (h5p-interactive-book.js:10)
    at Object.i.newRunnable (frame.bundle.js:3)
    at HTMLDivElement.<anonymous> (frame.bundle.js:3)
    at Function.each (frame.bundle.js:3)
    at init.each (frame.bundle.js:3)
    at Object.i.init (frame.bundle.js:3)
    at HTMLDocument.<anonymous> (frame.bundle.js:3)
    at u (frame.bundle.js:3)
    at Object.fireWith [as resolveWith] (frame.bundle.js:3)
    at Function.ready (frame.bundle.js:3)
```

I'm not a Javascript expert, but it seems like babel-polyfill has been used in both h5p-standalone and in the above h5p content type. Babel-polyfill states that it should [not be used in libraries](https://github.com/babel/babel-loader/issues/401/) for this reason. 

Does anyone have any idea where babel-polyfill is being used in h5p-standalone? I am willing to prepare a pull request to deal with this. I think babel-polyfill should either be removed as a dependency (as it is already being used by H5P where needed) or there should be some kind of option for it.

**Comment by mikedawson** [comment]
H5P Standalone is not including or using babel-polyfill. The problem is that somehow H5PStandalone has decided to load the interactive book Javascript twice (which uses babel-polyfill, which then causes the error):

![main](https://user-images.githubusercontent.com/4280622/94709266-96fa7800-0356-11eb-8cd6-ba193e57936c.png)

**Comment by contrerasjose** [comment]
Hi, still not solved?
Thanks for your comments

**Comment by elmitxel** [comment]
this problem is not solved yet, right?
Thanks for your comments

**Comment by AnimMouse** [comment]
Any news for this?

**Comment by mimidotsuser** [comment]
> Any news for this?

@busgurlu is already doing some work that will fix the issue here #54

**Comment by bryandonmarc** [comment]
Seems this issue hasn't been resolved yet. I went ahead and tried building https://github.com/tunapanda/h5p-standalone/pull/102, but there was an error. Fixed it in https://github.com/tunapanda/h5p-standalone/pull/105.

Building this, then copying the output of the `dist` folder and [using it directly](https://github.com/tunapanda/h5p-standalone#direct-use) makes it work perfectly.
